.github/workflows/ci.yml

```diff
@@ -82,7 +82,7 @@ jobs:
 
   test_profiling:
     name: Test profiling
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Don't ask me why GitHub chose to have a latest != the actual latest Ubuntu, doesn't make sense to me.